build.gradle

```diff
@@ -107,41 +107,60 @@ ext.libraries = [
 
 group = 'com.linkedin.venice'
 
+publishing {
+  repositories {
+    mavenLocal()
+    maven {
+      name "LinkedInJFrog"
+      url "https://linkedin.jfrog.io/artifactory/venice"
+      credentials {
+        if (System.getenv('JFROG_USERNAME') != null && System.getenv('JFROG_API_KEY') != null) {
```

Comment:
Should this be checked outside of publishing block?

Reply:
I can't move it outside of publishing block as it would prevent publishing to `mavenLocal` too. It is useful to have that target for testing changes to maven publishing. I can try moving it outside of the `credentials` block - but it can't go any further up.

Also, if the credentials are not configured, the publishing will fail due to auth errors. Is there any issue to keeping it as it is currently set?

Reply:
Moved one level up. Let me know if that is okay